src/Consts.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as Types from "./Types";
+
+export const TabsterAttributeName = "data-tabster" as const;
+export const TabsterDummyInputAttributeName = "data-tabster-dummy" as const;
+
+export const FocusableSelector = [
+    "a[href]",
+    "button:not([disabled])",
+    "input:not([disabled])",
+    "select:not([disabled])",
+    "textarea:not([disabled])",
+    "*[tabindex]",
+    "*[contenteditable]",
+    "details > summary",
+    "audio[controls]",
+    "video[controls]",
+].join(", ");
+
+export const AsyncFocusSources: Types.AsyncFocusSources = {
+    EscapeGroupper: 1,
+    Restorer: 2,
+    Deloser: 3,
+} as const;
+
+export const ObservedElementAccesibilities: Types.ObservedElementAccesibilities =
+    {
+        Any: 0,
+        Accessible: 1,
+        Focusable: 2,
+    } as const;
```

Comment:
```suggestion
export const ObservedElementAccesibilities: Types.ObservedElementAccesibilities =
    {
        Any: 0,
        Accessible: 1,
        Focusable: 2,
    };
```

you've already typed this so there's no need to export as const

Reply:
Not really: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions — check the enum-like patterns section about const assertions.

Reply:
![image](https://github.com/microsoft/tabster/assets/28045781/89857272-dd71-4f60-a88a-9a135b135967)
![image](https://github.com/microsoft/tabster/assets/28045781/ee9149aa-18d4-4f8f-94bb-0d91f6504bf0)

Reply:
but in the typings you can do equivalent with `readonly` 

https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAZiEMC8MDeAoG2YEMBcMAjADRY4BGhATGTjMIQMxkC+eED40GGoksCrgBOhAGIB5CSnTlsBYnUo0l2RjBYZWvAJZgoAU2FxcwAzEnTM9YQdwATcABsAnnkJEA3HJi2HztyoYam8bO0cwVwZmb20MIWEAOlwUZFRqDAQQZNT03gSc1CZvLMKNIA

Reply:
alternatively instead of declaring this twice you can do

import {  ObservedElementAccessibilities as  ObservedElementAccessibilitiesValues }  from './Const'

export type ObservedElementAccessibilities = typeof ObservedElementAccessibilitiesValues

Reply:
Makes sense, adjusted.